Source/HOHQMesh.f90

```diff
@@ -141,10 +141,11 @@ SUBROUTINE HOHQMesh(projectDict, project, stats, didGenerate3DMesh, test)
             PRINT *, "*******************"
             PRINT *, "2D Mesh Statistics:"
             PRINT *, "*******************"
-            PRINT *, "   Total time         = ", stopWatch % elapsedTime(TC_SECONDS)
-            PRINT *, "   Number of nodes    = ", project % mesh % nodes    % COUNT()
-            PRINT *, "   Number of Edges    = ", project % mesh % edges    % COUNT()
-            PRINT *, "   Number of Elements = ", project % mesh % elements % COUNT()
+            PRINT *, "   Total time             = ", stopWatch % elapsedTime(TC_SECONDS)
+            PRINT *, "   Number of nodes        = ", project % mesh % nodes    % COUNT()
+            PRINT *, "   Number of Edges        = ", project % mesh % edges    % COUNT()
+            PRINT *, "   Number of Elements     = ", project % mesh % elements % COUNT()
+            PRINT *, "   Number of Subdivisions = ", project % numberOfLevelsUsed
```

Comment:
No, this is only written to the output file and is not used in the benchmarks. It could be used, but the other data that is used (New, Nodes, etc.) provides more detail. I added it here because it seems interesting to know how much the background grid is subdivided, now that it is checked.